ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template3TestData.kt

```diff
@@ -62,31 +62,10 @@ internal val TestData.template3: PaywallData
                             "Cancel anytime before if you didn't love our app.",
                         iconID = "attachment",
                     ),
-                    PaywallData.LocalizedConfiguration.Feature(
-                        title = "Today",
-                        content = "Full access to 1000+ workouts plus free meal plan worth {{ price }}.",
-                        iconID = "tick",
-                    ),
-                    PaywallData.LocalizedConfiguration.Feature(
-                        title = "Day 7",
-                        content = "Get a reminder about when your trial is about to end.",
-                        iconID = "notification",
-                    ),
-                    PaywallData.LocalizedConfiguration.Feature(
-                        title = "Day 14",
-                        content = "You'll automatically get subscribed. " +
-                            "Cancel anytime before if you didn't love our app.",
-                        iconID = "attachment",
-                    ),
-                    PaywallData.LocalizedConfiguration.Feature(
-                        title = "Day 14",
-                        content = "You'll automatically get subscribed. " +
-                            "Cancel anytime before if you didn't love our app.",
-                        iconID = "attachment",
-                    ),
```

Comment:
Have you tested it with a bunch of features?

Reply:
Yeah it still works well. I just wanted this to look the same as the mock ups / iOS so we can compare the most common use case of features being vertically centered 😇